setup/install_gatling/40_post_install/60_restart_server.rb

```diff
@@ -1,6 +1,10 @@
 test_name 'Restart puppet master to pick up configuration changes'
 
-service_name = get_puppet_server_service_name_from_env()
```

Comment:
This didn't seem to return the right value for me, it was also the first issue I had and just put in a dumb check rather than try to fix this method. I assume fixing this method would be a much better way to resolve this and all the other places I check for install type.